app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteListFragment.kt

```diff
@@ -130,7 +130,11 @@ open class AutocompleteListFragment : Fragment() {
     }
 
     override fun onPrepareOptionsMenu(menu: Menu?) {
-        menu?.findItem(R.id.remove)?.isVisible = isSelectionMode() || domainList.adapter.itemCount > 1
+        val removeItem = menu?.findItem(R.id.remove)
+
+        removeItem?.isVisible = isSelectionMode() || domainList.adapter.itemCount > 1
```

Comment:
I see, this is actually the same R.id.remove being used for (the different) menus of both fragments (List and Remove), and the "prepareMenu" logic is being shared between them, so it's a little more more complex but does work :)